.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,3 +17,7 @@ _Use a TODO when your pull request is Work in Progress._
 
 - [ ] 
 - [ ] 
+
+## Testbot runs (Filled in by Devs)
+- [] KVM x86:
```

Comment:
You need a space inside these brackets iirc, for the markdown checkbox to show ok.
Also the template misses an empty newline at the end.

Reply:
What about the test VBox as well?